okapi-core/src/main/java/org/folio/okapi/managers/ModuleManager.java

```diff
@@ -90,7 +95,7 @@ private void invalidateCacheEntry(String id) {
   /**
    * Load the modules from the database, if not already loaded.
    *
-   * @param fut
+   * @param fut future
```

Comment:
This line should be deleted because it does not give any additional information.